Misanthropic-Mistletoes/reactive-nutshell
# Caroline three

# Description

Edit works on Events

Fixes # (issue)

New Feature

Please delete options that are not relevant.


# Testing Instructions

1. Commit changes to your branch.
2. git checkout master
3. git fetch --all
4. git checkout caroline-three
5. run code
6. run npm start
7. click edit on a single event, the data should populate into an edit form, and submit will save changes, rendering updated event to the DOM

# Checklist:
